# 12/2 ~ Mootha & Subramanian ~ Gene Set Enrichment Analysis
## Literature Review
  Both Mootha’s paper “PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes” and Subramanian’s paper “Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles” focus on the research application of gene set enrichment analysis (GSEA). The former, published first, is a more biological experiment focused analysis that discusses the GSEA technique with respects to how it identifies subtle differences in the RNA expression profiles of diabetic human muscle tissue that could not be explained by differential gene expression alone. The latter paper simultaneously builds on and establishes the GSEA technique by painting the complete picture of their statistical approach, specifically they describe the mathematics for the normalization of enrichment scores and multiple hypothesis testing. In addition to fleshing out technique, they also Subramanian evaluates the effectiveness of GSEA against a variety of biological datasets such as p53 cancer cell lines, acute leukemias, and a meta-analysis of lung cancer datasets. Through diverse experimentation they establish GSEA as a multifaceted tool for the evaluation of differential expression results.
	
  It cannot be understated just how impactful GSEA as a tool has been in the evaluation of RNA sequencing data. One may argue that their methodology has inspired multiple branches of gene set driven analysis focused on understanding gene sets in the context of experimentation and learning them from sample expression profiles. These papers provide a stable foundation not just because of the intrinsic value of their methods, but also because of the quality of their writing. Many statistical approaches and be marred in the complexity of the mathematics used to derive them; however, the authors of these papers go for a leaner approach more focused on exploring their methods application. They understand that when trying to advertise methods to experimentalists outside of the domains of applied mathematics, you must establish the value of your product based on its versatility and ease of application in the problem space affecting your end users. The only small issue I have is with the latter paper. The former paper includes a great graphic describing the GSEA method visually. I think reusing that graphic in their methods paper would really help readers who were not aware of the prior publication. Regardless, both papers effectively inform readers about the method and provide room for speculation on its application.
## Questions
1.	The GSEA method is a startlingly simple approach that leverages computational methods mixed with biological foundations to produce a modern cornerstone of bioinformatics analysis. Given the increase in the number of bioinformaticians today, do you believe that there are simple methods still left to be discovered today?
2.	These papers were released in the early 2000s and have seen consistent use and popularity since then. Modern methods that leverage machine learning driven approaches do not see anywhere near the same momentum as these methods did upon release. Given the challenges with setting up and executing machine learning tools, do you think it is wise to pursue machine learning methods over these simple classical analysis methods?
Mootha, Vamsi K., et al. "PGC-1α-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes." Nature genetics 34.3 (2003): 267-273.
Subramanian, Aravind, et al. "Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles." Proceedings of the National Academy of Sciences 102.43 (2005): 15545-15550.
